models/amazon_ads__ad_report.sql

```diff
@@ -51,10 +51,10 @@ fields as (
         portfolios.portfolio_name,
         portfolios.portfolio_id,
         campaigns.campaign_name,
-        campaigns.campaign_id,
+        campaigns.campaign_id, 
         ad_groups.ad_group_name,
-        ad_groups.ad_group_id,
-        ads.ad_id,
+        ad_groups.ad_group_id, 
+        ads.ad_id, 
```

Comment:
We found in the other ad reporting packages that id's can actually erroneously be deleted within the history models. As such, when possible we want to be sure we select the id's from the `report`. Would you be able to go through the end models and update where the id fields are being selected from.

If they exist in the report, let's use the id from the report. If not, then we can use the history record.

Reply:
updated